[DOC-770] Update xCluster documentation for architecture limitations section (#27782)
yugabyte/yugabyte-db: docs/content/preview/api/ysql/exprs/sequence_functions/func_nextval.md

Use the `nextval( sequence_name )` function to return the next value from the sequence cache for the current session. If no more values are available in the cache, the session allocates a block of numbers for the cache and returns the first one. The number of elements allocated is determined by the `cache` option specified as part of the `CREATE SEQUENCE` statement.

**Limitations**

> - xCluster replication except for transactional automatic mode is not compatible with sequences and therefore is not compatible with this feature.

Reviewer: ```suggestion
- xCluster replication (unless you are using transactional automatic mode) is not compatible with sequences and therefore is not compatible with this feature.
```

Author: done
